index.html

```diff
@@ -0,0 +1,111 @@
+          <li class="navbar-item"><a href="#home">Upcomig classes</a></li>
+        </ul>
+        <button class="sign-up"><a href="#">SignUp</a></button>
+        </div>
+      </nav>
+      <div class="home-body" id="home">
+        <p class="body-disc">"Hello! Future Web Developers"</p>
+        <h1 class="home-heading">WE ARE HERE TO TRAIN THE BEST WEBSITE<span> DEVELOPERS FOR THE MARKET</span> IN 2023.</h1>
+        <p class="body-disc2">We provide the best suitable teaching method that can turn anyone to be the best in the market. And we also provide a flexable study time.</p>
+        <h2 class="heading2">Registration for 2023 is Open<span> (Feb16) ~ (Feb21)</span></h2>
+      </div>
+    </header>
+  </section>  
+
+  <section class="about" id="about">
+    <h2 class="about-heading">Our Teaching Methods</h2>
+    <div class="heading-under"></div>
+    <div class="about-list">
+      <div class="about-item">
+        <i class="fa-solid fa-person-chalkboard"></i>
+        <h3 class="about-item-title">Lecture</h3>
+        <p class="about-item-disc">We provide a live lecture classes by the best teachers in the topic.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-bars-progress"></i>
+        <h3 class="about-item-title">Projects</h3>
+        <p class="about-item-disc">During the program there are lots of real world projets for practical purposes.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-user-group"></i>
+        <h3 class="about-item-title">Pair Coding</h3>
+        <p class="about-item-disc">We will pair you with another student to stay motivated and also to learn from each other.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-code"></i>
+        <h3 class="about-item-title">Code reviews</h3>
+        <p class="about-item-disc">Every project you submit will be reviewed senior developers inorder to make you sharp.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-people-pulling"></i>
+        <h3 class="about-item-title">Mentor</h3>
+        <p class="about-item-disc">In Every steps you are about to take we will make sure you get the most appropriate consling.</p>
+      </div>
+    </div>
+    <button class="about-btn">Register Now</button>
+    <div class="about-link">
+      <a href="#">See The Whole Program</a>
+    </div>
+  </section>
+
+  <div class="section-feature">
+
+  </div>
+
+  <section class="sponsers" id="Scholarship">
+    <div class="sponsers-heading">
+      <h3 class="sponsers-title">Partner</h3>
+      <div class="under"></div>
+    </div>
+    <ul class="sponsers-list">
+      <li class="sponser-item"><a >mozilla</a></li>
+      <li class="sponser-item"><a>Google</a></li>
+      <li class="sponser-item"><a>NAVER</a></li>
+      <li class="sponser-item"><a>daumkakao</a></li>
+      <li class="sponser-item"><a>airbnb</a></li>
+    </ul>
+  </section>
+
```

Comment:
currently, there is an overflow of items in the partner section in the mobile version. please fix this to improve the UX of your webpage

![image](https://user-images.githubusercontent.com/60612329/219875418-eb740c09-f67a-4b11-9740-1185c0c063e3.png)